lib/power_action_utils.pm

```diff
@@ -154,6 +154,12 @@ sub poweroff_x11 {
         send_key "alt-d";              # shut_d_own
         assert_screen 'logout-confirm-dialog', 10;
         send_key "alt-o";              # _o_k
+
+        if (check_screen('shutdown-auth', 10)) {
```

Comment:
Instead of `check_screen` can we not check that the system hasn't actually shut down? Thanks to @OleksandrOrlov we have `check_shutdown` in the testapi, so probably the following can just work:

```perl
if (!check_shutdown()) {
    …
```

Reply:
Sorry, I can't follow you here? I need to check if it wants a password, not if it is shut down